src/config/mod.rs

```diff
@@ -98,19 +98,34 @@ pub enum OutputState {
     Mirroring(String),
 }
 
-fn default_enabled() -> OutputState {
+fn default_state() -> OutputState {
     OutputState::Enabled
 }
 
+#[derive(Debug, Deserialize, Serialize, Clone, Copy, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum AdaptiveSync {
+    #[serde(rename = "true")]
+    Enabled,
+    #[serde(rename = "false")]
+    Disabled,
+    Force,
+}
```

Comment:
It seems a little odd to use a different terminology (enabled/force vs automatic/always) from the protocol, though it doesn't really matter that much.